pocs/observatory.py

```diff
@@ -734,8 +736,20 @@ def _create_scheduler(self):
                 module = load_module(
                     'pocs.scheduler.{}'.format(scheduler_type))
 
+                obstruction_list = self.config['location'].get('obstructions', list())
+                default_horizon = self.config['location'].get('horizon', 30.)
+
+                horizon = horizon_utils.Horizon(
+                    obstructions=obstruction_list,
+                    base_horizon=default_horizon
+                )
+
                 # Simple constraint for now
-                constraints = [MoonAvoidance(), Duration(30 * u.deg)]
+                constraints = [
+                    Altitude(horizon=horizon),
```

Comment:
For now this will just create a flat line at the base horizon level so shouldn't affect current scheduler.